### PR TITLE
Add viewIsDescendantOf for UIManager on Android

### PR DESCRIPTION
**Pull request by Swordsman-Inaction**
Add the ability for UIManager to check if a node is an ancestor of anther one on Android like #7876 did on iOS

### CODE REVIEW

**Comment by Swordsman-Inaction** [comment]
Any one to review this PR?

**Comment by Swordsman-Inaction** [comment]
@janicduplessis Do you know who can review this PR?

File: ReactAndroid/src/main/java/com/facebook/react/uimanager/UIImplementation.java [hunk 1]
```diff
@@ -470,6 +470,22 @@ public void findSubviewIn(int reactTag, float targetX, float targetY, Callback c
   }
 
   /**
+   *  Check if the first shadow node is the descendant of the second shadow node
+   */
+  public void viewIsDescendantOf(
+          final int reactTag,
```
Comment (janicduplessis) [inline ReactAndroid/src/main/java/com/facebook/react/uimanager/UIImplementation.java]: Nit: 4 space continuation to match the style of the file.

File: ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java [hunk 2]
```diff
@@ -411,6 +411,17 @@ public void findSubviewIn(
   }
 
   /**
+   *  Check if the first shadow node is the descendant of the second shadow node
+   */
+  @ReactMethod
+  public void viewIsDescendantOf(
+          final int reactTag,
```
Comment (janicduplessis) [inline ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java]: Nit: Same here

**Comment by Swordsman-Inaction** [comment]
@mkonicek Can you review?

**Comment by Swordsman-Inaction** [comment]
@janicduplessis Hey, I must say the review progress is very slow.

**Comment by ms88privat** [comment]
Please :)

**Comment by janicduplessis** [comment]
@Swordsman-Inaction Sorry about the delay, can you rebase on master to get tests passing then I'll merge.

**Comment by Swordsman-Inaction** [comment]
@janicduplessis Done

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Comment by Swordsman-Inaction** [comment]
Something is wrong with the github bot?

**Comment by janicduplessis** [comment]
@facebook-github-bot shipit

**Comment by janicduplessis** [comment]
@hramos Could you try shipping this

**Comment by hramos** [comment]
Will do - let me first see if we can troubleshoot why the bot didn't work on this PR.

**Comment by Swordsman-Inaction** [comment]
@hramos Still not work?
It doesn't work since April 25. 

image

I thought I could make this available in 0.44. 😞

**Comment by ericnakagawa** [comment]
Trying to land this today.

**Comment by Swordsman-Inaction** [comment]
Any update?

**Pull request closed without merging**